wsdl/ver10/schema/onvif.xsd

```diff
@@ -6563,12 +6563,6 @@ If set to 0.0, infinity will be used.</xs:documentation>
 	</xs:complexType>
 	<!--===============================-->
 	<xs:complexType name="ItemList">
-		<xs:annotation>
-			<xs:documentation>
-			List of parameters according to the corresponding ItemListDescription.
```

Comment:
Any reason why we chose to remove this line? 

As per our discussion, the concerning requirement was 'Each item in the list shall have a unique name.' which was to be removed.

Reply:
It's not wrong but type annotations are completely hidden. Any annotation should go into elements or attributes. See definition in tt:Config
```
<xs:element name="Parameters" type="tt:ItemList">
  <xs:annotation>
    <xs:documentation>List of configuration parameters as defined in the corresponding description.</xs:documentation>
  </xs:annotation>
</xs:element>
```